src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -249,6 +276,96 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
         return funPrefixNamer("enumerators", makeNameStyle(this.enumeratorNamingStyle, legalizeName));
     }
 
+    protected makeNamesForPropertyGetterAndSetter(
+        _c: ClassType,
+        _className: Name,
+        _p: ClassProperty,
+        _jsonName: string,
+        name: Name
+    ): [Name, Name, Name] {
+        const getterName = new DependencyName(this._memberNamingFunction, name.order, lookup => `get_${lookup(name)}`);
+        const mutableGetterName = new DependencyName(this._memberNamingFunction, name.order, lookup => `getMutable_${lookup(name)}`);
+        const setterName = new DependencyName(this._memberNamingFunction, name.order, lookup => `set_${lookup(name)}`);
+        return [getterName, mutableGetterName, setterName];
+    }
+
+    protected makePropertyDependencyNames(
+        c: ClassType,
+        className: Name,
+        p: ClassProperty,
+        jsonName: string,
+        name: Name
+    ): Name[] {
+        const getterAndSetterNames = this.makeNamesForPropertyGetterAndSetter(c, className, p, jsonName, name);
+        this._gettersAndSettersForPropertyName.set(name, getterAndSetterNames);
+        return getterAndSetterNames;
+    }
+
+    protected startFile(basename: Sourcelike, includeHelper: boolean = true): void {
+        assert(this._currentFilename === undefined, "Previous file wasn't finished");
+        if (basename !== undefined) {
+            this._currentFilename = `${this.sourcelikeToString(basename)}`;
```

Comment:
My comment here was not about using `sourcelikeToString`, it was about the string interpolation.

```
`${this.sourcelikeToString(basename)}`
```

is equivalent to

```
this.sourcelikeToString(basename)
```

Reply:
Oooh aok, thanks!